ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -81,15 +81,40 @@ tests:
   container:
     from: test-bin
 - as: subsystem-aws
+  cluster_claim:
+    achitecture: amd64
+    cloud: aws
+    owner: dpp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.8"
   steps:
-    cluster_profile: aws
+    pre:
+    - as: expose-ci-images
```

Comment:
Why not use the `ref` ?

Reply:
b/c the ref does not work when KUBECONFIG is set to cluster under test (=always in jobs running on claims). That's why I unset KUBECONFIG few lines below. I think I can just add the unset to the ref and use that, but I wanted to test if this alone will be enough to make the job work.

Reply:
aaah!